roms_tools/setup/cdr_forcing.py

```diff
@@ -503,136 +285,227 @@ def plot_volume_flux(self, start=None, end=None, releases="all"):
             Start datetime for the plot. If None, defaults to `self.start_time`.
         end : datetime or None
             End datetime for the plot. If None, defaults to `self.end_time`.
-        releases : str, list of str, or "all", optional
-            A string or list of release names to plot.
+        release_names : list of str, or "all", optional
+            A list of release names to plot.
             If "all", the method will plot all releases.
             The default is "all".
+
+        Raises
+        ------
+        ValueError
+            If self.releases are not of type VolumeRelease.
+            If `release_names` is not a list of strings or "all".
+            If any of the specified release names do not exist in `self.releases`.
         """
 
+        if self.release_type != VolumeRelease:
+            raise ValueError(
+                "plot_volume_flux is only supported when all releases are of type VolumeRelease."
+            )
+
         start = start or self.start_time
         end = end or self.end_time
 
-        # Handle "all" releases case
-        if releases == "all":
-            releases = [k for k in self.releases if k != "_tracer_metadata"]
-        # Validate input for release names
-        self._validate_release_input(releases)
+        valid_release_names = [r.name for r in self.releases]
+
+        if release_names == "all":
```

Comment:
Consider creating a global constant (or ClassVar) for "all", e.g. `INCLUDE_ALL_RELEASE_NAMES="all"`. I see it repeated twice in this method and it might be nice to have more context about what it means (and so we can re-use it on line 328, 364, 380, 395, etc)

Reply:
Done in [ed368b1](https://github.com/CWorthy-ocean/roms-tools/pull/301/commits/ed368b16584b9ff66b3fba2ae92ad620d414e7ae).